Tests/Unit/TokenProvider/ContentLocaleProviderTest.php

```diff
@@ -24,7 +25,8 @@ public function setUp()
         parent::setUp();
 
         $this->uriContext = $this->prophesize('Symfony\Cmf\Component\RoutingAuto\UriContext');
-        $this->provider = new ContentLocaleProvider();
+        $this->slugifier = $this->prophesize('Symfony\Cmf\Bundle\CoreBundle\Slugifier\SlugifierInterface');
```

Comment:
we never called slugifier reveal as there is no slugifier needed for ContentLocaleProvider. Please revert these changes